### PR TITLE
Post applicant and residence to api

### PR DESCRIPTION
**Pull request by lavanyareddyreddy**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
-no

#### Which User story does this relate to?
- #148091485, #148091521

#### Brief feature description
- post appliant and put residence data to API

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
-no

#### JS errors diff as a result of this PR?
- no

### CODE REVIEW

**Comment by lavanyareddyreddy** [comment]
retest

File: app/controllers/rfa/a01_controller.rb [hunk 1]
```diff
@@ -23,24 +23,34 @@ def edit
...
+    if applicants.present?
+      applicants_result = []
+
+      applicants.each do |applicant|
+        applicant = applicant.permit!
+        applicants_result << rfa_applicant_helper.create(params[:id], applicant.to_json)
+      end
+
+      application_response[:applicants] = applicants_result
+      @applicant_id = application_response[:applicants][0].id if application_response[:applicants].present?
```
Comment (ratneshraval) [inline app/controllers/rfa/a01_controller.rb]: Let's remove this line.

File: app/controllers/rfa/a01_controller.rb [hunk 2]
```diff
@@ -23,24 +23,38 @@ def edit
 
     @residence_types =  dictionaries_helper.residence_ownership_types
     @ethnicity_types = dictionaries_helper.ethnicity_types
+    # @race_types = dictionaries_helper.race_types
     @address_types = dictionaries_helper.address_types
     @relationship_types = dictionaries_helper.relationship_types
+
+    @application = rfa_application_helper.find_by_id(params[:id])
+    # @application.applicants = rfa_application_helper.find_applicants()
+    @application.applicants = rfa_applicant_helper.find_by_application_id(params[:id])
```
Comment (ratneshraval) [inline app/controllers/rfa/a01_controller.rb]: This is good but another option for this section is to do
- rfa_application_helper.applicants(application_id)
- rfa_application_helper.residence(application_id)

Comment (lavanyareddyreddy) [inline app/controllers/rfa/a01_controller.rb]: good suggestion, but lets leave it the way it is now. what do you think?

Comment (ratneshraval) [inline app/controllers/rfa/a01_controller.rb]: 👍

**Pull request merged**